### ISSUE TITLE
Account tests failing in Travis

### ISSUE THREAD

**Issue by jzeferino** [massivedisaster/ADAL#81]
Apparently some Account tests are failing in Travis, but we can't replicate in our local machines.

Here is one of the reports.

https://travis-ci.org/massivedisaster/ADAL/builds/239879073?utm_source=github_status&utm_medium=notification

Can you guys help solve it?

**Comment by Andrer757** [comment]
The problem is that the tests fail randomly since whenever you make rebuild the answer is different ... I can comment them as I did with the other, but we don't even know if there will be another test that is not from the accounts that will fail.
Sometimes gives an exception, sometimes fails a test, sometimes passes without stress

**Comment by jzeferino** [comment]
@PedroOkawa Can you take a closer look at this?

**Comment by jzeferino** [comment]
This is the updated information in this:

##Temporary fix in #88.

1. I've downgraded de launched avd te api 19 since there is a [problem](https://github.com/travis-ci/travis-ci/issues/6340) with the 24 version. We could try to bypass keylock as better alternative.

2. Commented the `testClearAccounts` since the assert is failing. There is some possibility of a bug e junit. They have issue calling assert inside callbacks (i've read this somewere but can't find it.)

3. I've also added the skin for the emulator since it was taking to much too launch.

Common travis errors:

```
java.lang.RuntimeException: Waited for the root of the view hierarchy to have window focus and not be requesting layout for over 10 seconds. If you specified a non default root matcher, it may be picking a root that never takes focus. Otherwise, something is seriously wrong. Selected Root:
	Root{application-window-token=android.view.ViewRootImpl$W@392936e0, window-token=android.view.ViewRootImpl$W@392936e0, has-window-focus=false, layout-params-type=1, layout-params-string=WM.LayoutParams{(0,0)(fillxfill) sim=#100 ty=1 fl=#1010100 pfl=0x8 wanim=0x1030461 surfaceInsets=Rect(0, 0 - 0, 0)}, decor-view-string=DecorView{id=-1, visibility=VISIBLE, width=320, height=480, has-focus=false, has-focusable=true, has-window-focus=false, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}}
```
and 

```
junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:48)
Tests on test(AVD) - 5.0 failed: Instrumentation run failed due to 'junit.framework.AssertionFailedError'
:sample:connectedDebugAndroidTest FAILED
```

**Comment by jzeferino** [comment]
closed by 27e849c.
